usaspending_api/etl/es_mapper.json

```diff
@@ -0,0 +1,121 @@
+{
+  "settings": {
+    "index.mapping.ignore_malformed": true,
+    "index.max_result_window" : 500000
```

Comment:
@miahunsicker There is another ES json, `es_settings.json`, in this PR specifically for configuration. Do we need these settings here for every data ingest or can they be moved to the setting file?